styles/gemini/catppuccin.user.css

```diff
@@ -0,0 +1,235 @@
+    --bard-color-surface-container-highest: lighten(@base, 5%);
+    --bard-color-surface-container-high: @surface0;
+    --bard-color-response-container-flipped-background: @surface0;
+    --bard-color-inverse-on-surface: darken(@mantle, 1%);
+    --bard-color-sidenav-mobile-controls: @text;
+    --bard-color-outline-variant: @subtext0;
+    --bard-color-main-container-background: @base;
+    --bard-color-main-container-background-rgb: #rgbify(@base) [];
+    --bard-color-window-container-background: @base;
+    --bard-color-response-container-background: @base;
+    --bard-color-response-container-background-rgb: #rgbify(@base) [];
+    --bard-color-selected-conversation-metadata-v2: fade(@accent-color, 50%);
+    --bard-color-onhover-conversation-metadata-button-v2: @surface1;
+    --bard-color-selected-conversation-metadata-button-v2: fade(
+      @accent-color,
+      40%
+    );
+    --bard-color-new-conversation-button: @surface0;
+    --bard-color-on-new-conversation-button: @text;
+    --bard-color-onhover-conversation-metadata-v2: @surface0;
+    --bard-color-on-surface: @text;
+    --bard-color-on-surface-variant: @subtext1;
+    --bard-color-sidenav-upgrade-button: @surface0;
+    --bard-color-form-field-placeholder: @subtext0;
+    --bard-color-extensions-window-container-background: @crust;
+    --bard-color-extension-card-container-background: @mantle;
+    --bard-color-discovery-carousel-background: @mantle;
+    --bard-color-example-prompt-background: @crust;
+    --bard-color-bard-mode-logo-prefix: @subtext1;
+    --bard-color-discovery-carousel-icon: @accent-color;
+    --bard-color-neutral-96: @crust;
+    --bard-color-adv-marketing-basic-background: @base;
+    --bard-color-adv-marketing-product-background: @surface0;
+    --bard-color-adv-marketing-disclaimer: @subtext0;
+    --bard-color-input-area-buttons-selected-background: fade(
+      @accent-color,
+      70%
+    );
+    --bard-color-regular-banner: @accent-color;
+    --bard-color-regular-banner-text: @crust;
+    --bard-color-surface-dim-rgb: #rgbify(@crust) [];
+    --bard-color-on-surface-rgb: #rgbify(@subtext0) [];
+    --bard-color-surface: @crust;
+    --bard-color-skeleton-loader-background-1-rgb: #rgbify(@mantle) [];
+    --bard-color-skeleton-loader-background-2-rgb: #rgbify(@accent-color) [];
+    --bard-color-skeleton-loader-background-3-rgb: #rgbify(@accent-color) [];
+    --bard-color-brand-text-gradient-stop-1: @blue;
+    --bard-color-brand-text-gradient-stop-2: @sapphire;
+    --bard-color-brand-text-gradient-stop-3: @peach;
+    --bard-color-on-surface-opacity-38: fade(@teal, 25%);
+    --bard-color-on-surface-opacity-12: fade(@teal, 10%);
+    --bard-color-footer-background: @mantle;
+
+    --bard-color-code-quotes-and-meta: @peach;
+    --bard-color-code-variables: @red;
+    --bard-color-code-literal: @yellow;
+    --bard-color-code-string: @green;
+    --bard-color-code-class: @yellow;
+    --bard-color-code-keyword: @mauve;
```

Comment:
```suggestion
```

See https://github.com/catppuccin/userstyles/blob/main/docs/tips-and-tricks.md#how-do-i-theme-code-blocks--syntax-highlighting. They use highlight.js on Gemini.

Reply:
And this one @Dandraghas.